.github/workflows/ci.yml

```diff
@@ -13,10 +13,71 @@ on:
       - release/**
 
 jobs:
+# add get-diff job 
+  get-diff:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      output-contract: ${{ steps.contract.outputs.run }}
+      output-certora-contract: ${{ steps.certora-contract.outputs.run }}
+      output-scripts: ${{ steps.scripts.outputs.run }}
+      output-agents: ${{ steps.agents.outputs.run }}
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get diff for contract
+        uses: technote-space/get-diff-action@v4.0.1
+        id: get-diff-contract # via outputs
+        with:
+          PATTERNS: |
+            +(contracts)/*.sol
+
+      - id: contract
+        run: echo "::set-output name=run::true"
+        if: steps.get-diff-contract.outputs.diff
```

Comment:
As mentioned previously, don't use the diff step's output to create another step's output, just export the diff step's output directly.

Reply:
Done